package.json

```diff
@@ -13,6 +13,7 @@
     "prebuild.prod": "npm run prebuild",
     "build.prod": "npx ng build --progress false --prod",
     "build": "npx ng build --progress false",
+    "license": "license-checker  --excludePackages dockstore-ui2@2.3.0 --csv > THIRD-PARTY-LICENSES.csv",
```

Comment:
Does this work if you do `npm run license`? I could be wrong, but unless you've installed license globally, I think you need to precede this with npx.

Reply:
Yeah, just tried it again and it worked with `npm run license`

Reply:
I don't understand why, but I tried it as well and it worked. I guess package.json scripts have an implied npx or something.

Reply:
The way I see it, if there's a global license-checker and a local one, npx will ensure the local one runs.

Reply:
Hmm, interesting, when you do `npm run`, it apparently automatically adds `node_modules/.bin` to your path. Never knew. Ideally you would still use npx for Gary's case, but not crucial.